zenodo/modules/profiles/templates/zenodo_profiles/orcid_profile.html

```diff
@@ -0,0 +1,221 @@
+{#-
```

Comment:
We should check if it's possible to unify these "profile" templates to some hierarchy in order to avoid this much duplicity in code. Eg. both views have a "Popular Uploads" title and some search results, this should be either extracted in some separate template or be a Jinja macro maybe...